scripts/test/scan-build.sh

```diff
@@ -63,13 +64,15 @@ scan-build \
 
 # Make this not get scanned for dead assignments, since it has lots of
 # dead assignments we don't care about.
+# shellcheck disable=SC2086
 scan-build \
     $CHECKERS \
     -disable-checker deadcode.DeadStores \
     make -j5 -k ./src/ext/ed25519/ref10/libed25519_ref10.a
 
+# shellcheck disable=SC2086
 scan-build \
-    $CHECKERS $OUTPUTARG \
+    $CHECKERS "$OUTPUTARG" \
```

Comment:
Are we sure this is correct? The way we set `OUTPUTARG` above seems to depend on word splitting between the `-o` and `$SCAN_BUILD_OUTPUT`.

Reply:
Fixed in da264f7c766b332f596a92766f7625c4a17abf70.